test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```
@@ -189,125 +192,139 @@
+  (.getThreadPool (create-server-with-partial-http-config config)))
+
+(defn get-thread-pool-for-default-server
+  []
+  (.getThreadPool (Server.)))
+
+(defn default-server-max-threads
+  []
+  (.getMaxThreads (get-thread-pool-for-default-server)))
+
+(defn get-max-threads-for-partial-http-config
+  [config]
+  (.getMaxThreads (get-thread-pool-for-partial-http-config config)))
+
+(deftest create-server-max-threads-test
+  (testing "default max threads passed through to thread pool"
+    (is (= (default-server-max-threads)
+           (get-max-threads-for-partial-http-config {:max-threads nil}))))
+  (testing "custom max threads passed through to thread pool"
+    (is (= 9042
+           (get-max-threads-for-partial-http-config
+             {:max-threads 9042, :queue-max-size nil})))))
+
+(deftest create-server-queue-max-size-test
+  (let [get-queue-for-partial-http-config (fn [config]
+                                            (get-server-thread-pool-queue
+                                              (create-server-with-config
+                                                (munge-http-connector-config
+                                                  config))))
+        default-server-min-threads        (.getMinThreads
+                                            (get-thread-pool-for-default-server))]
+    (testing "default queue max size passed through to thread pool queue"
+      (is (= (.getMaxCapacity (get-server-thread-pool-queue (Server.)))
+             (.getMaxCapacity (get-queue-for-partial-http-config
+                                {:queue-max-size nil})))))
+    (testing "custom default queue max size passed through to thread pool queue"
+      (is (= 393
+             (.getMaxCapacity (get-queue-for-partial-http-config
+                                {:queue-max-size 393})))))
+    (testing (str "default max threads passed through to thread pool when "
+                  "queue-max-size set")
+      (is (= (default-server-max-threads)
```

This looks like a bug - looks like accidental parens around `default-server-min-threads` - that seems like it should throw an error since `default-server-min-threads` should be an integer, right?